params/config.go

```diff
@@ -419,6 +431,12 @@ func (c *ChainConfig) checkCompatible(newcfg *ChainConfig, head *big.Int) *Confi
 	if isForkIncompatible(c.KoreCompatibleBlock, newcfg.KoreCompatibleBlock, head) {
 		return newCompatError("Kore Block", c.KoreCompatibleBlock, newcfg.KoreCompatibleBlock)
 	}
+	if isForkIncompatible(c.Kip103CompatibleBlock, newcfg.Kip103CompatibleBlock, head) {
```

Comment:
I thought this was left intentionally blank because kip103 is an optional fork. @aidan-kwon is this ok?